protocol/x/clob/keeper/process_operations.go

```diff
@@ -19,6 +19,25 @@ import (
 	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
 )
 
+// fetchOrdersInvolvedInOpQueue fetches all OrderIds involved in an operations
+// queue's matches + short term order placements and returns them as a set.
+func fetchOrdersInvolvedInOpQueue(
+	operations []types.InternalOperation,
+) (orderIdSet map[types.OrderId]struct{}) {
+	orderIdSet = make(map[types.OrderId]struct{})
+	for _, operation := range operations {
+		if shortTermOrderPlacement := operation.GetShortTermOrderPlacement(); shortTermOrderPlacement != nil {
+			orderId := shortTermOrderPlacement.GetOrder().OrderId
+			orderIdSet[orderId] = struct{}{}
+		}
+		if clobMatch := operation.GetMatch(); clobMatch != nil {
```

Comment:
I added all the order ids involved in a match, but technically there were no errors in `dev` without this statement. 
I think technically we can remove this since I think it's an invariant that every short term order Id that shows up in the any opqueue matches must appear before as a short term order placement. Additionally, there would be no short term order placements that show up without being involved in a match.

What do you think?